dotnet/test/Autogen.Ollama.Tests/OllamaClientAgentTests.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// OllamaClientAgentTests.cs
+
+using System.Text.Json;
+using AutoGen.Core;
+using FluentAssertions;
+
+namespace Autogen.Ollama.Tests;
+
+public class OllamaClientAgentTests
+{
+    private readonly OllamaClientAgent _ollamaClientAgent;
+    private readonly string _host = "http://localhost:11434";
+
+    public OllamaClientAgentTests()
+    {
+        var httpClient = new HttpClient
+        {
+            BaseAddress = new Uri(_host),
+            Timeout = TimeSpan.FromSeconds(250)
+        };
+        _ollamaClientAgent = new OllamaClientAgent(httpClient, "TestAgent", "mixtral:latest");
+    }
+
+    [Fact]
```

Comment:
These tests will fail on pipeline because there won't be ollama instance running

You can use `ApiKeyFact` with environment variable to skip a test when that env variable doesn't exist.

Here's an example
https://github.com/microsoft/autogen/blob/11a43421e349113f699b9b706e6515d7023bf879/dotnet/test/AutoGen.Mistral.Tests/MistralClientTests.cs#L22

Reply:
you have all the reason :) my mistake